driver/http.go

```diff
@@ -84,6 +86,59 @@ func respDecode(body io.ReadCloser, v interface{}) error {
 	return nil
 }
 
+type metadata struct {
+	CreatedAt *time.Time `json:"created_at,omitempty"`
+	UpdatedAt *time.Time `json:"updated_at,omitempty"`
+	DeletedAt *time.Time `json:"deleted_at,omitempty"`
+}
+
+func newRespMetadata(m *metadata) *api.ResponseMetadata {
+	r := &api.ResponseMetadata{}
+	if m.CreatedAt != nil {
+		r.CreatedAt = timestamppb.New(*m.CreatedAt)
+	}
+	if m.UpdatedAt != nil {
+		r.UpdatedAt = timestamppb.New(*m.UpdatedAt)
```

Comment:
it will also be having createdAt.

Reply:
All three are parsed, I didn't get the comment.